docs/developer-guide/firstanalysis.md

```diff
@@ -0,0 +1,144 @@
+# Your first analysis
+
+This is a tutorial exercise for writing analyses in Goblint.
+We will implement a very simple sign analysis.
+You need to install goblint for development.
+
+## First test
+
+Create a C file in the Goblint root directory for testing.
+Here we call it `example.c`. This is the first test:
+
+
+```
```

Comment:
The code blocks should specify language (`c`/`ocaml`). I think it uses some generic tokenizer and highlighter without them (or tries to guess the language?).